console-extensions.json

```diff
@@ -57,26 +57,22 @@
     }
   },
   {
-    "type": "console.navigation/href",
+    "type": "console.navigation/resource-ns",
     "properties": {
       "id": "templates",
       "section": "virtualization",
       "name": "%plugin__kubevirt-plugin~Templates%",
-      "href": "/templates",
-      "prefixNamespaced": true,
+      "resource": "virtualmachinetemplates",
```

Comment:
is there a reason for using `"resource"` instead of `"model"` property?
if it should be `"resource"`, should it be `"virtualmachinetemplates"` and not simply `"templates"`?

Reply:
This is a bit of a hack and not sure about that cc: @vojtechszocs . We are using a custom page to show the virtual machine templates (`console.navigation/href`) . It's not a resource page like virtual machines or pods, jobs, etc (`console.navigation/resource-ns`). But the nav item of these resources has a different namespace mechanism (that fixes the bug if applied also to the templates)

Reply:
Template is a resource, we simply want to filter only the Templates resource with a VM object when we fetch them, I think we should follow same logic as we do on VM or VMI, @yaacov wdyt?

Reply:
a. Is it better to use "resource": "templates", @vojtechszocs WDYT ?
b. can we use "console.page/resource/list" and "console.page/resource/details" to override an internal resource view ?